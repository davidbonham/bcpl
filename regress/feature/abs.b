GET "libhdr"

LET start() BE $(
    LET cases = TABLE -5, -1, 0, 1, 5
    LET a = -5
    LET b = ABS a
    newline()
    writehex(ABS #X8000000000000000, 16)
    newline()
    writehex(ABS #XFFFFFFFFFFFFFFFF, 16)
    newline()
    newline()
    FOR i = 0 TO 4 DO $(
        writen(ABS cases!i)
        newline()
    $)
$)
