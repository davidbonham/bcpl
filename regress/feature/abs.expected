-- stdout -------------------------------------------

8000000000000000
0000000000000001

5
1
0
1
5

-- stderr -------------------------------------------

